scania-digital-design-system/sdds
# fix/standardise storybook control option and default options

**Describe pull-request**  
This PR standardises the controls and default options for all components.

**Solving issue**  
Fixes: [AB#2483](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2483)

**How to test**  
1. Go to storybook link below
2. Check in all components and try out the controls.

### NOTE
This PR also replaces svgs with out own icons, and gives the user the option of having either a webcomponent or native one.
